index.html

```diff
@@ -2,95 +2,167 @@
-Interested in starting a project?  
-Let’s talk:
-
-Enter email
-
-We'll never share your email with anyone else.
-
-New York
-123 Lane
-Suite 100
-Albany, NY 12345
-202 555 0144
-
-Florida
-Ocean Drive
-Suite 201
-Orlando, FL 22345
-502 555 0144
-
-California
-Mountain Street
-Suite 105
-San Diego, CA 22345
-702 555 0144
+<!-- Menu Content  -->
+<header class="main-navigation">
+	<div class="container nav-container">
+		<h1 class="logo-text">S&amp;J</h1>
+		<img class="menu" src="img/nav-hamburger.png" alt="menu button">
+		<nav class="nav">
+			<h1 class="logo-text logo-overlay">S&amp;J</h1>
+			<a class="nav-link" href="#"><img class="menu close" src="img/nav-hamburger-close.png" alt="menu button"></a>
+			<a class="nav-link" href="index.html">Home</a>
+			<a class="nav-link" href="services.html">Services</a>
+			<a class="nav-link" href="contact.html">Contact</a>
+		</nav>
+	</div>
+</header>
+
+<!-- Home Page Content: -->
+<div class="container home">
+	<div class="main-header">
+		<img class="jumbotron" 
+			srcset = "img/home/home-mobile-jumbotron.png 500w,
+					img/home/home-jumbotron.png 1024w"
+			sizes = "(max-width: 500px) 500px,
+					1024px" 
+			src="img/home/home-jumbotron.png" alt="Photo of a silver, modern style building.">
+			<h2>Integrity,<br> Excellence,<br>Progress.</h2>
+	</div>
+	<section class="content-section">
+		<div class="text-content">
+			<h4>Smith &amp; Jones Architects</h4>
+			<p>Et sed autem causae appareat, tempor abhorreant te mei, facer facilisis sit ea. Eu timeam vidisse consectetuer sed. Duo etiam
+			laboramus dissentiet in, nec no errem</p>
+			<div class="btn">
+				<a href="#">Learn More</a>
+			</div>
+		</div>
+		<div class="img-content">
+			<img
+				srcset = "img/home/home-mobile-img-1.png 500w,
+						 img/home/home-img-1.png 1024w"
+				sizes = "(max-width: 500px) 500px,
+						1024px"
+				src = "img/home/home-img-1.png" alt="Photo of blueprints">
+		</div>
+	</section>
+	<section class="content-section inverse-content">
+		<div class="img-content">
+			<img
+				srcset = "img/home/home-mobile-img-2.png 500w,
+						img/home/home-img-2.png 1024w"
+				sizes = "(max-width: 500px) 500px,
+						1024px"
+				src="img/home/home-img-2.png" alt="Photo of a building's reflection in water.">
+		</div>
+		<div class="text-content">
+			<h4>Futuristic Designs</h4>
+			<p>Et sed autem causae appareat, tempor abhorreant te mei, facer facilisis sit ea. Eu timeam vidisse consectetuer sed. Duo etiam
+			laboramus dissentiet in, nec no errem</p>
+			<div class="btn">
+				<a href="#">View Designs</a>
+			</div>
+		</div>
+	</section>
+	<section class="content-stacked">
+		<h4>Recent Projects</h4>
+		<img 
+			srcset = "img/home/home-mobile-villas-img.png 500w,
+					img/home/home-villas-img.png 1024w"
+			sizes = "(max-width: 500px) 500px,
+					1024px"
+			src="img/home/home-villas-img.png" alt="Second slide">
+		<h3 class="section-title one">THE VILLAS</h3>
+		<div class="text-content">
```

Comment:
Try to avoid just using numbers for class names. They're too hard to associate to content. If you have to you can do something like project-1, etc.